ftw/notification/base/browser/views.py

```diff
@@ -132,7 +133,8 @@ def users(self):
                 if member is None:
                     continue
 
-                user = dict(title=t.title,
+                title = t.title if ITitledTokenizedTerm.providedBy(t) else t.value
```

Comment:
Well that's the problem. I couldn't reproduce the error. So i don't know for sure.
I suspect though it is the same as the other terms. `title` is an optional field: https://github.com/starzel/plone.principalsource/blob/master/plone/principalsource/term.py#L12